pkg/file/splitter/internal/job.go

```diff
@@ -167,16 +157,19 @@ func (s *SimpleSplitterJob) sumLevel(lvl int) ([]byte, error) {
 		}
 	}
 
-	s.hasher.Reset()
-	err = s.hasher.SetSpanBytes(c[:8])
+	hasher := bmtpool.Get()
+
+	err = hasher.SetSpanBytes(c[:8])
 	if err != nil {
 		return nil, err
 	}
-	_, err = s.hasher.Write(c[8:])
+	_, err = hasher.Write(c[8:])
 	if err != nil {
 		return nil, err
 	}
-	ref := s.hasher.Sum(nil)
+	ref := hasher.Sum(nil)
+	bmtpool.Put(hasher)
```

Comment:
Same issue with leaking resource.

Reply:
@acud this comment is from previous review, same as the other (you fixed), where the `hasher` is not returned to the 'bmtpool' if one of these two error occur.

Reply:
sorry missed this. fixed :+1: